ppml/trusted-bigdata/spark-executor-template.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - name: spark-executor
+    env:
+      - name: ATTESTATION
+        value: false
+      - name: ATTESTATION_URL
+        value: your_attestation_url
+      #- name: PCCS_URL
+      #  value: your_pccs_url
+      #- name: ATTESTATION_ID
+      #  valueFrom:
+      #    secretKeyRef:
+      #      name: kms-secret
+      #      key: app_id
+      #- name: ATTESTATION_KEY
+      #  valueFrom:
+      #    secretKeyRef:
+      #      name: kms-secret
+      #      key: app_key
+    volumeMounts:
+      - name: device-plugin
+        mountPath: /var/lib/kubelet/device-plugins
+      - name: aesm-socket
+        mountPath: /var/run/aesmd/aesm.socket
+      - name: nfs-storage
+        mountPath: /ppml/data
+      - name: nfs-storage
+        mountPath: /root/.kube/config
+        subPath: kubeconfig
+    resources:
+      requests:
+        cpu: 16
+        memory: 128Gi
+        sgx.intel.com/epc: 133258905600
```

Comment:
133258905600 is too big, can we set it as the same size of the epc mem?

Reply:
fixed